docs/dictionary/command/filter.lcdoc

```diff
@@ -71,20 +71,17 @@ Description:
 Use the <filter> command to pick specific lines, items, keys or elements
 in a container or expression.
 
-The <filter>...with form and the <filter>...matching form retain the
+The '<filter>...with' form and the '<filter>...matching' form retain the
```

Comment:
@seaniepie could you use back ticks to make these render as code. Would need to drop the `<>` around filter as that causes the parser to mess with the word.